kernel/README.md

```diff
@@ -24,15 +24,87 @@ in the kernel source code. You can modify it as needed.
 After the kernel source code is ready it is possible to build the kernel.
 
 ```bash
-./build-kernel.sh build
+$ ./build-kernel.sh build
 ```
 
 
 ## Install the Kernel in the default path for Kata
 
 Kata Containers uses some default path to search a kernel to boot. To install
 on this path, the following command will install it to the default Kata
-containers path.
+containers path.  
+
 ```bash
-./build-kernel.sh install
+$ ./build-kernel.sh install
+```
+
+## Summit Kernel Changes
+
+Kata Containers packaging repository holds the kernel configs and patches. The
+config and patches could be for many versions but we only make sure to test the
+kernel version defined on the [runtime versions file][runtime-versions-file]
+
+## How is it tested.
+
+Kata Containers CI scripts will install the kernel from packages or builder
+from sources.
+
+If kernel defined in [runtime versions file][runtime-versions-file] is build
```

Comment:
"If the kernel defined in the [runtime versions file][runtime-versions-file] is built ..."